src/function/aggregate/nested/histogram.cpp

```diff
@@ -101,23 +101,30 @@ static void HistogramCombineFunction(Vector &state, Vector &combined, idx_t coun
 	}
 }
 
+static inline ValidityMask &AnyValidity(Vector &v) {
+	return (v.GetVectorType() == VectorType::CONSTANT_VECTOR) ? ConstantVector::Validity(v) : FlatVector::Validity(v);
```

Comment:
This seems like it might cause problems since constant and flat vectors are semantically different. `FlatVector::Validity` used to do this switch, but we changed it since it hid certain types of bugs. `Finalize` should generally be called with the `result` set to a flat vector as that is the expected usage of the API (and function APIs in general).